### PR TITLE
[#170865103] Rspec setup

### PR DESCRIPTION
**Pull request by Durgesh00**
Setup rspec and other gems required for testing and authentication like faker, apitome , simplecov, jwt for the project

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -44,5 +47,9 @@ group :development do
   gem "spring-watcher-listen", "~> 2.0.0"
 end
 
+group :test do
+  gem 'simplecov', require: false
+end
 # Windows does not include zoneinfo files, so bundle the tzinfo-data gem
 gem "tzinfo-data", platforms: %i[mingw mswin x64_mingw jruby]
+gem 'jwt'
```
Comment (smriti154) [inline Gemfile]: Remove this, jwt token will be in separate PR

File: config/database.yml [hunk 2]
```diff
@@ -1,4 +1,4 @@
-# PostgreSQL. Versions 9.1 and up are supported.
+# PostgreSQL. Versions 8.2 and up are supported.
```
Comment (smriti154) [inline config/database.yml]: database.yml file doesn't get commited

File: app/models/organization.rb [hunk 3]
```diff
@@ -0,0 +1,15 @@
+# frozen_string_literal: true
+
+class Organization < ApplicationRecord
+  after_create :create_tenant
```
Comment (smriti154) [inline app/models/organization.rb]: Add the model validations for Organization model

File: test/models/organisation_test.rb [hunk 4]
```diff
@@ -0,0 +1,9 @@
+# frozen_string_literal: true
```
Comment (smriti154) [inline test/models/organisation_test.rb]: Remove this file

File: test/fixtures/organisations.yml [hunk 5]
```diff
@@ -0,0 +1,11 @@
+# Read about fixtures at http://api.rubyonrails.org/classes/ActiveRecord/FixtureSet.html
```
Comment (smriti154) [inline test/fixtures/organisations.yml]: Remove this file

Comment (Durgesh00) [reply test/fixtures/organisations.yml]: done

File: test/models/organisation_test.rb (hunk 4 above)
Comment (Durgesh00) [reply test/models/organisation_test.rb]: done

File: config/database.yml (hunk 2 above)
Comment (Durgesh00) [reply config/database.yml]: done

File: Gemfile (hunk 1 above)
Comment (Durgesh00) [reply Gemfile]: done

File: app/models/organization.rb (hunk 3 above)
Comment (Durgesh00) [reply app/models/organization.rb]: done

**Review by smriti154 — APPROVED** [review]
